.github/ISSUE_TEMPLATE/remove_package.yml

```diff
@@ -0,0 +1,34 @@
+name: Remove Package(s)
+description: Remove one or more existing packages from the Swift Package Index.
+title: 'Remove <Package>'
+labels: ['Remove Package']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        We understand that you may wish to remove a package from the Swift Package Index, and we respect that wish.
+        
+        Note that we automatically track packages which are a dependency of another package on the index, so your package may re-appear.
+
+        The Swift Package Index will show when a project has been archived, and will automatically update it's information if it's renamed or moved to a new owner.
```

Comment:
```suggestion
        The Swift Package Index will show when a project has been archived, and will automatically update its information if it's renamed or moved to a new owner.
```

Reply:
failed English 4 times 😂  committed, but I know Dave will come through and reword all of this anyway!